lib/Parser/Parse.cpp

```diff
@@ -2436,19 +2436,26 @@ bool Parser::IsImportOrExportStatementValidHere()
 }
 
 template<bool buildAST>
-ParseNodePtr Parser::ParseImportDeclaration()
+ParseNodePtr Parser::ParseImport()
 {
     Assert(m_scriptContext->GetConfig()->IsES6ModuleEnabled());
     Assert(m_token.tk == tkIMPORT);
 
+    m_pscan->Scan();
```

Comment:
Yes, updated in 2nd iteration.

---
In reply to: [114901862](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r114901862) [](ancestors = 114901862)